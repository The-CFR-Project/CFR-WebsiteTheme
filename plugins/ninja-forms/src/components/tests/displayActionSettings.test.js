import { render, cleanup } from '@testing-library/react';
import { action } from './mockData';
import { DisplayActionSettings } from '../';

describe( 'submissions-display-action-settings', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const { container } = render(
			<DisplayActionSettings value={ action } />
		);
		expect( container ).toMatchSnapshot();
	} );
} );
