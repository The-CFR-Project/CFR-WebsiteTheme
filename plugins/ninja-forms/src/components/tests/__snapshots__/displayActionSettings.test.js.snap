// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-display-action-settings Matches snapshot 1`] = `
<div>
  <ul>
    <li>
      <b>
        label
      </b>
      : 
      Email Confirmation 2
    </li>
    <li>
      <b>
        to
      </b>
      : 
      {wp:user_email},{wp:user_email}
    </li>
    <li>
      <b>
        email_subject
      </b>
      : 
      Submission Confirmation 
    </li>
  </ul>
</div>
`;
